python/openmldb/sdk/sdk.py

```diff
@@ -25,27 +25,45 @@
 sys.path.append(os.path.dirname(__file__) + "/..")
 logger = logging.getLogger("openmldb_sdk")
 
-class OpenmldbSdkOptions(object):
+class OpenmldbClusterSdkOptions(object):
     def __init__(self, zk_cluster, zk_path, session_timeout = 3000):
         self.zk_cluster = zk_cluster
         self.zk_path = zk_path
         self.session_timeout = session_timeout
 
+class OpenmldbStandaloneSdkOptions(object):
+    def __init__(self, host, port):
+        self.host = host
+        self.port = port
+
 class OpenmldbSdk(object):
-    def __init__(self, options):
+    def __init__(self, options, is_cluster_mode):
+        self.is_cluster_mode = is_cluster_mode
         self.options = options
         self.sdk = None
 
     def init(self):
-        options = sql_router_sdk.SQLRouterOptions()
-        options.zk_cluster = self.options.zk_cluster
-        options.zk_path = self.options.zk_path
-        self.sdk = sql_router_sdk.NewClusterSQLRouter(options)
-        if not self.sdk:
-            logger.error("fail to init openmldb sdk with zk cluster %s and zk path %s"%(options.zk_cluster, options.zk_path))
-            return False
-        logger.info("init openmldb sdk done with zk cluster %s and zk path %s"%(options.zk_cluster, options.zk_path))
-        return True
+        if self.is_cluster_mode:
+            options = sql_router_sdk.SQLRouterOptions()
+            options.zk_cluster = self.options.zk_cluster
+            options.zk_path = self.options.zk_path
+            self.sdk = sql_router_sdk.NewClusterSQLRouter(options)
+            if not self.sdk:
+                logger.error("fail to init openmldb sdk with zk cluster %s and zk path %s"%(options.zk_cluster, options.zk_path))
+                return False
+            logger.info("init openmldb sdk done with zk cluster %s and zk path %s"%(options.zk_cluster, options.zk_path))
+            return True
```

Comment:
`return True` can be reused for both modes.

Reply:
updated.